stdlib/http/src/main/ballerina/src/http/auth/authz_handler.bal

```diff
@@ -28,6 +28,10 @@ public type AuthzHandler object {
     public cache:Cache? positiveAuthzCache;
     public cache:Cache? negativeAuthzCache;
 
+    # Initializes the `AuthzHandler` object.
+    #
+    # + positiveAuthzCache - `Cache` instance, which is cache positive authorizations
+    # + negativeAuthzCache - `Cache` instance, which is cache negative authorizations
```

Comment:
```suggestion
    # + negativeAuthzCache - `Cache` instance including negative authorizations
```

Reply:
@praneesha, this comment, gives out the wrong meaning. We need to say something like, 'this is the cache, which keeps negative authorizations'.